hack/godep-save.sh

```diff
@@ -33,35 +35,35 @@ REQUIRED_BINS=(
   "./..."
 )
 
+STAGING_REPOS=(
```

Comment:
Can this be dynamic like https://github.com/kubernetes/kubernetes/blob/master/hack/update-staging-godeps.sh#L81 instead of a static list?